modules/cnf-configuring-workload-hints.adoc

```diff
@@ -30,4 +31,53 @@
 [NOTE]
 ====
 When the `realTime` workload hint flag is set to `true` in a performance profile, add the `cpu-quota.crio.io: disable` annotation to every guaranteed pod with pinned CPUs. This annotation is necessary to prevent the degradation of the process performance within the pod. If the `realTime` workload hint is not explicitly set then it defaults to `true`.
-====
\ No newline at end of file
+====
+
+The following table describes how combinations of power consumption and real-time settings impact on latency.
+
+.Impact of combinations of power consumption and real-time settings on latency
+[cols="1,1,1,1",options="header"]
+|===
+|Performance Profile creator setting |Hint |Environment |Description
+
+|Default
+a|[source,terminal]
```

Comment:
This PR is specifically trying to avoid content changes unrelated to the structure. #72761 should indeed be the place that addresses this, we just need to watch out for merge conflicts.